PowerShellSyntax.tmLanguage

```diff
@@ -9,7 +9,7 @@
 		<string>psd1</string>
 	</array>
 	<key>name</key>
-	<string>PowerShell</string>
+	<string>Powershell</string>
```

Comment:
Hmmm, was this necessary to get VS Code to operate correctly?  I wonder if it'd have a negative effect on other editors?